test/cli/index.js

```diff
@@ -156,6 +157,18 @@ describe('cli', () => {
     })
   })
 
+  describe('db.json -m postbody-middleware.js', () => {
+    beforeEach((done) => {
+      child = cli([ dbFile, '-m', middlewareFiles.postbody ])
+      serverReady(PORT, done)
+    })
+
+    it('Should have post body in middleware', (done) => {
```

Comment:
I've updated the line, and also updated the code to the new standards. Is it okay to get accepted now? I'd love to have this in the next release.